WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+Reminders.swift

```diff
@@ -1,10 +1,159 @@
 import UIKit
 import SwiftUI
 
+class BloggingRemindersTracker {
```

Comment:
I think there's enough here that we could probably move it to its own file.

Reply:
Yeah, this is actually a mistake.  I created the class here for convenience, but forgot to move it out.

Reply:
Moved out to a new file of its own.